### DISCUSSION TITLE
Misuse of the tool

### DISCUSSION THREAD

**Discussion by nicolefallen** [ThioJoe/YT-Spammer-Purge#552] [General]
Hi!

I saw LTT’s vid on this and watched ThioJoe’s after that. What a fantastic initiative! I was immediately enamoured!

One possible issue to consider, however, is the ability for people to misuse this tool to mass-report non-spammers, either because they’re actually angry at someone, or just to cause havoc. Since users can be anonymous even when creating a GCP account, and certainly when creating a YouTube account, it would seem a small thing to do create new credentials, misuse the tool, then dump the account. This could cause very unhappy commenters or make a lot of unnecessary work for channel owners.

I don’t know enough about the APIs or the solution to suggest a good mitigation, but maybe you all who know it well can think of something smart. Of course, I hope it won’t be an issue at all, but knowing the Internet….

**Comment by Kurdakov** [comment]
you might launch the program and see that it's not really good for misuse,
one problem though is not in utility, but how youtube treats reports.
say bots, which repeat others comments should be marked as such,  it looks like youtube filters in this case train network to delete good comments (so there is apparent problem - that many good comments are deleted), but then again - it's due to youtube inflexibility,  not a problem of YT Spammer Purge itself or people who report spam on repeated comments.

**Reply by Firecul** [reply]
> you might launch the program and see that it's not really good for misuse,

Yeah it takes work to alter the program for that purpose, by default the only modes available to issue reports are the auto and sensitive modes. The rest only work on channels you have permissions for already.

**Comment by KendallDoesCoding** [comment]
The only way someone could misuse this tool if its not their video is by reporting not spam comments right? Youtube, does a review of those comments, idk how often, but if its wasting youtube's employees time (which it wont, that's their job), they can just make a tool or improve their spam filtering 🤣
